### DISCUSSION TITLE
Dynamic resolution update support

### DISCUSSION THREAD

**Discussion by Sanjubisanal** [FreeRDP/FreeRDP#7336] [Q&A] [answered]
Does shadow server support dynamic resolution..?  
below are my server details.

In ubuntu 18.04, I am running the shadow server.

:~$ freerdp-shadow-cli -version
[11:12:13:322] [2747:2747] [INFO][com.freerdp.server.shadow] - FreeRDP version 2.2.0 (git n/a)

:~$freerdp-shadow-cli -auth

**Comment by akallabeth** [comment, marked as answer]
No. Shadow requires clients to use the same resolution as what is mirrored.
